Explorer/Assets/DCL/CharacterCamera/Systems/UpdateCursorInputSystem.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using DCL.CharacterCamera.Components;
+using DCL.Input;
+using DCL.Input.Systems;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace DCL.CharacterCamera.Systems
+{
+    [UpdateInGroup(typeof(InputGroup))]
+    public partial class UpdateCursorInputSystem : UpdateInputSystem<CameraInput, CameraComponent>
```

Comment:
You should add `UpdateBefore(typeof(UpdateCameraInputSystem))` as there is a dependency between systems